<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MOHAMED_GABIR" failures="1" tests="1" name="edgeTest" time="26.232" errors="0" timestamp="2025-03-16T23:45:48 GMT">
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="20.503">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- edgeTest -->
